cyences_app_for_splunk/bin/cs_product_list.py

```diff
@@ -320,28 +320,84 @@ def build_source_reviewer_search(by, values):
 | tstats values(host) as hosts where index=* sourcetype="*WinEventLog" source="*WinEventLog:Security" by source sourcetype index | eval sources = source." (".sourcetype.")" | stats count values(index) as index dc(hosts) as host_count by sources | stats count values(*) as * | eval sources=if(count>0,sources,"WinEventLog:Security (WinEventLog)")] | append [
 | tstats values(host) as hosts where index=* sourcetype="*WinEventLog" source="*WinEventLog:System" by source sourcetype index | eval sources = source." (".sourcetype.")" | stats count values(index) as index dc(hosts) as host_count by sources | stats count values(*) as * | eval sources=if(count>0,sources,"WinEventLog:System (WinEventLog)")] | append [
-| tstats values(host) as hosts where index=* source="powershell*" sourcetype="MSAD:*:Health" by source sourcetype index | eval sources = source." (".sourcetype.")" | stats count values(index) as index dc(hosts) as host_count by sources | stats count values(*) as * | eval sources=if(count>0,sources,"powershell (MSAD:*:Health)")] | append [
 | tstats values(host) as hosts where index=* sourcetype="ActiveDirectory" by sourcetype index | stats count values(index) as index dc(hosts) as host_count by sourcetype | stats count values(*) as * | eval sourcetype=if(count>0,sourcetype,"ActiveDirectory")  | rename sourcetype as sources]""",
                 "earliest_time": "-1d@d",
                 "latest_time": "now",
             }
         ],
     },
+    {
+        "name": "Windows AD",
+        "metadata_count_search": '| tstats count where index=* source IN ("WinEventLog:DFS Replication","WinEventLog:Directory Service","WinEventLog:Microsoft-AzureADPasswordProtection-DCAgent/Admin","PerfmonMk:DFS_Replicated_Folders") OR sourcetype IN ("MSAD:NT6:Netlogon","MSAD:NT6:Replication","MSAD:NT6:Health","MSAD:NT6:SiteInfo","Script:TimesyncConfiguration","Script:TimesyncStatus","windows:certstore:ca:issued") ',
+        "macro_configurations": [
+            {
+                "macro_name": "cs_windows_idx",
+                "label": "Windows AD Data",
+                "search": r"""| tstats count where index=* source="WinEventLog:DFS Replication" by source | rename source as sources | stats count | eval sources=if(count>0, sources, "WinEventLog:DFS Replication") | append [
```

Comment:
Can we simplify this into just two parts:

source IN (xyz, abc) .... | append [sourcetype IN (a, b, c) ....]
Something like this?